models/url.js

```diff
@@ -15,7 +15,8 @@ const urlSchema = new mongoose.Schema({
     short: {
         type: String,
         required: true,
-        default: nanoid(),
+        unique: true,
+        default: nanoid,
```

Comment:
Function reference is passed, and when new object is created the function will be called.